roles/reproducer/tasks/configure_controller.yml

```diff
@@ -107,53 +108,33 @@
         dest: "{{ _ctl_reproducer_basedir }}/parameters/interfaces-info.yml"
         content: "{{ cifmw_libvirt_manager_mac_map | to_nice_yaml }}"
 
-    # Here, we update the existing cifmw_virtualbmc_known_hosts fact
-    # to inject networking information such as provisioning MAC (if any),
-    # and fixed Boot mode
-    - name: Generate IPMI information
+    # Deploy Sushy Emulator on the Ansible controlelr,
```

Comment:
minor typo but we can fix it later... :)

Reply:
Thanks mate, I will fix this and a few other minor things in follow up <3